docs/Installation-Guide.rst

```diff
@@ -146,7 +146,19 @@ Only **Apple Clang** version 8.1 or higher is supported.
 
      git clone --recursive https://github.com/Microsoft/LightGBM ; cd LightGBM
      mkdir build ; cd build
+
+     # For Mojave
+     cmake \
+       -DOpenMP_C_FLAGS="-Xpreprocessor -fopenmp -I/usr/local/opt/libomp/include" \
```

Comment:
I think that 
```
-I"$(brew --prefix libomp)/include"
```
is better, because it shows the general case, from where this path comes from and how to customize it.

We provide all instructions for `brew`, so `brew` here is OK too.

Reply:
Unfortunately, subshell doesn't seem to work in subprocess 😢
https://gist.github.com/henry0312/81664a9710056f0d59136813ebff2651 (L.15, 16 and 54).

Reply:
Sorry, maybe I'm missing something, but my comment was about manual compilation of CLI version. Did you mean Python's subprocess?

Reply:
I'm sorry I misunderstood 🙇
I'll fix.

Reply:
No problem! Thank you very much for your brilliant work! 😃